shared/modules/persistence_module.go

```diff
@@ -38,7 +38,7 @@ type PersistenceRWContext interface {
 
 // NOTE: There's not really a use case for a write only interface,
 // but it abstracts and contrasts nicely against the read only context
-// TODO(andrew): convert address and public key to string not bytes; pocket/issues/163
+// TODO (andrew) convert address and public key to string not bytes #147
```

Comment:
I don't think 147 is the appropriate issue for it. `convert address and public key to string` should be a sepeate issue

Reply:
Do you want me to resolve this as part of https://github.com/pokt-network/pocket/issues/147?

Reply:
I think it's supposed to say #149